ev3dev2/sensor/lego.py

```diff
@@ -661,6 +675,61 @@ def wait_until_angle_changed_by(self, delta, direction_sensitive=False):
             while abs(start_angle - self.value(0)) < delta:
                 time.sleep(0.01)
 
+    def circle_angle(self):
+        """
+        As the gryo rotates clockwise the angle increases, it will increase
+        by 360 for each full rotation. As the gyro rotates counter-clockwise
+        the gyro angle will decrease.
+
+        The angles on a circle have the opposite behavior though, they start
+        at 0 and increase as you move counter-clockwise around the circle.
+
+        Convert the gyro angle to the angle on a circle. We consider the initial
```

Comment:
I don't understand this... what circle is involved here?

Reply:
A circle that represents the robot’s heading